alica_engine/src/engine/teammanager/TeamManager.cpp

```diff
@@ -28,29 +28,29 @@ TeamManager::~TeamManager()
 void TeamManager::init()
 {
     essentials::SystemConfig& sc = essentials::SystemConfig::getInstance();
-    this->teamTimeOut = AlicaTime::milliseconds(sc["Alica"]->get<unsigned long>("Alica.TeamTimeOut", NULL));
+    this->_teamTimeOut = AlicaTime::milliseconds(sc["Alica"]->get<unsigned long>("Alica.TeamTimeOut", NULL));
 
-    if (useConfigForTeam) {
+    if (_useConfigForTeam) {
         this->readTeamFromConfig();
     }
 }
 
 void TeamManager::readTeamFromConfig()
 {
     essentials::SystemConfig& sc = essentials::SystemConfig::getInstance();
-    std::string localAgentName = this->engine->getRobotName();
+    std::string localAgentName = this->_engine->getRobotName();
     std::shared_ptr<std::vector<std::string>> agentNames = sc["Globals"]->getSections("Globals.Team", NULL);
 
     Agent* agent;
```

Comment:
This declaration can move inside the loop

Reply:
used in log message outside loop

Reply:
...no, I meant agent. I fixed that quickly